app.js

```diff
@@ -1,15 +1,30 @@
+// State
+let volume = 10;
 
-// state
+// Helper functions
+function clamp(value, min, max) {
+    return Math.min(max, Math.max(min, value));
+}
```

Comment:
We ofter write little "mathy" helper functions like this as arrow functions with implicit returns:

```js
const clamp = (value, min, max) => Math.min(max, Math.max(min, value));
```

Reply:
Arguably, you could have made this more domain specific too:

```js
const clamp = (value, min, max) => Math.min(max, Math.max(min, value));
const clampVolume = volume => clamp(volume, 0, 20);
```

Reply:
Thanks! The implicit return thing is neat (in both senses). I don't like when I find myself creating a new block statement for one liners because it looks noisy.